pkg/controllers/restore/controller.go

```diff
@@ -273,20 +275,32 @@ func (h *handler) OnRestoreChange(_ string, restore *v1.Restore) (*v1.Restore, e
 	return restore, err
 }
 
-func (h *handler) restoreCRDs(created map[string]bool, objFromBackupCR ObjectsFromBackupCR) error {
+func (h *handler) restoreCRDs(created map[string]bool, objFromBackupCR ObjectsFromBackupCR) (crdsWithSubStatus []string, err error) {
 	for crdInfo, crdData := range objFromBackupCR.crdInfoToData {
 		err := h.restoreResource(crdInfo, crdData, false)
 		if err != nil {
-			return fmt.Errorf("restoreCRDs: %v", err)
+			return crdsWithSubStatus, fmt.Errorf("restoreCRDs: %v", err)
 		}
 		created[crdInfo.ConfigPath] = true
+
+		specs := crdData.Object[specMapKey].(map[string]interface{})
```

Comment:
nit: here would be a good place for comment describing why we care about the subresource